### PR TITLE
Add READMEs

### PR DESCRIPTION
**Pull request by vitalikda**

### CODE REVIEW

File: packages/grafbase-urql-exchange/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+2. Create URQL client
+
+```ts
+// client.ts
+import { cacheExchange, createClient, dedupExchange, fetchExchange } from 'urql'
+
+export const client = createClient({
+  url: process.env.GRAFBASE_API_URL,
+  fetchOptions: {
+    headers: { 'x-api-key': process.env.GRAFBASE_API_KEY }
```
Comment (notrab) [inline packages/grafbase-urql-exchange/README.md]: Frontend examples should use `Authorizatiom` and not mention the use of API keys.

File: packages/grafbase-apollo-link/README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+// client.ts
+import { HttpLink, split } from '@apollo/client'
+import { getOperationAST } from 'graphql'
+import { isLiveQuery, SSELink } from '@grafbase/apollo-link'
+
+export const createApolloLink = () => {
+  const sseLink = new SSELink({
+    uri: process.env.GRAFBASE_API_URL,
+    headers: {
+      'x-api-key': process.env.GRAFBASE_API_KEY
```
Comment (notrab) [inline packages/grafbase-apollo-link/README.md]: `Authorization` keys only

**Review by notrab — CHANGES_REQUESTED** [review]
Looks great! Covers mostly everything we discussed.

We should however not mention `x-api-key` since API keys currently are global access all. 

We could in the examples do something like this:

```ts
const token = '...'

const sseLink = new SSELink({
  uri: process.env.GRAFBASE_API_URL,
  headers: {
    'authorization': `Bearer ${token}`
  }
})
```

File: packages/grafbase-urql-exchange/README.md (hunk 1 above)
Comment (notrab) [reply packages/grafbase-urql-exchange/README.md]: Also, `Authorization` must be sent inside of the query params and not headers.

Comment (vitalikda) [reply packages/grafbase-urql-exchange/README.md]: fixed

File: packages/grafbase-apollo-link/README.md (hunk 2 above)
Comment (vitalikda) [reply packages/grafbase-apollo-link/README.md]: fixed

File: packages/grafbase-urql-exchange/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+```ts
+// client.ts
+import { cacheExchange, createClient, dedupExchange, fetchExchange } from 'urql'
+
+export const client = createClient({
+  url: process.env.GRAFBASE_API_URL,
+  fetchOptions: {
+    headers: {
+      authorization: `Bearer ${token}`
```
Comment (notrab) [inline packages/grafbase-urql-exchange/README.md:28]: Question: Do we read from `fetchOptions` inside the URQL exchange so we pass it along as a query param?

File: packages/grafbase-urql-exchange/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+```
+npm install urql graphql @grafbase/urql-exchange
+```
+
+2. Create URQL client
+
+```ts
+// client.ts
+import { cacheExchange, createClient, dedupExchange, fetchExchange } from 'urql'
```
Comment (notrab) [inline packages/grafbase-urql-exchange/README.md:19]:
```suggestion
import { cacheExchange, createClient, dedupExchange, fetchExchange } from 'urql'

// Token generated by your auth provider: https://grafbase.com/docs/reference/directives#auth
const token = '....'
```

Makes sense to include a placeholder

File: packages/grafbase-urql-exchange/README.md (hunk 3 above)
Comment (vitalikda) [reply packages/grafbase-urql-exchange/README.md:28]: Yes, `@grafbase/urql-exchange` accepts all headers and transforms them into query params

Comment (notrab) [reply packages/grafbase-urql-exchange/README.md:28]: Great. Thanks for clarifying.

**Review by notrab — APPROVED** [review]

**Pull request merged**